src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -263,13 +265,25 @@ def acr_queue(cmd,
         result = LongRunningOperation(cmd.cli_ctx)(client_registries.queue_build(
             build_request=build_request, resource_group_name=resource_group_name, registry_name=registry_name))
 
-        print("Queued a build with build-id: {}.".format(result.build_id))
-
         if no_logs == False:
-            print("Starting to stream the logs...")
+            size = os.path.getsize(FILE)
```

Comment:
I'm assuming all this logic should be put above this call near `queue_build` - no context is being passed at this point, it's already over

Reply:
The original work item is to combine the four print messages:
“Start to archive...
The source code tar ball file size...
Queue a build with us...
Start to stream logs...” 
into one single messsge:
“Sending the build context...”
Therefore, I am assuming it is an AND logic. The final condensed message should appear exactly at the place where “Starting to stream logs...” gets printed.